docs/whatsnew/5.2.rst

```diff
@@ -46,6 +47,34 @@ cosmologies with their non-flat equivalents.
     True
 
 
+.. _whatsnew-5.2-coordinates:
+
+Topocentric ITRS Frame
+======================
+
+A topocentric ITRS frame has been added that makes dealing with near-Earth objects
+easier and more intuitive.::
+
+    t = Time('J2010')
```

Comment:
Do we doctest what's new code snippets? If so, it needs `>>>` for doctest to pick up.

Reply:
Sure, I can add those.

Reply:
What about the comments and blank lines? Do I need to remove those?

Reply:
Never mind. I found a working example that includes these.